src/redux/actions/index.js

```diff
@@ -6,6 +6,7 @@ import {
   SET_CURRENT_CATEGORY,
   SET_LEARNT_PHRASES,
 } from '../constants';
+import {getAllCategories} from '../../data/dataUtils';
```

Comment:
To prevent a name conflict here, consider importing it as something else. How about this?
```suggestion
import {getAllCategories as getAllLocalCategories} from '../../data/dataUtils';
```